FHIR_README.md

```diff
@@ -0,0 +1,78 @@
+![img](./public/images/openemr-rest-api.png)
+
+### Overview
+
+Easy-to-use JSON-based REST API for OpenEMR FHIR. All code is done in classes and separate from the view to help with codebase modernization efforts. 
+
+### Prerequisite
+Enable this FHIR service in OpenEMR menu: Administration->Globals->Connectors->"Enable FHIR Provider Client Service" Select HAPI FHIR or Smart on FHIR, then Save
+
+
+### Endpoints
+ FHIR endpoints Use `http://localhost:8300/apis/fhir as base URI.`
+
+_Example:_ `http://localhost:8300/apis/fhir/Patient` returns a Patient's bundle resource, etc
+
+#### POST /api/auth
+
+Obtain an FHIR token with your login (returns an FHIR token). 
+Scope must match a site that has been setup in OpenEMR in the /sites/ directory.  If you haven't created additional sites
+then 'default' should be the scope.
+
+```sh
+curl -X POST -H 'Content-Type: application/json' 'http://localhost:8300/apis/fhir/auth' \
+-d '{
+    "grant_type":"password",
+    "username": "ServiceUser",
+    "password": "password",
+    "scope":"site id"
+}'
+```
+
+Response:
+```json
+{
+    "token_type": "Bearer",
+    "access_token": "eyJ0b2tlbiI6IjAwNmZ4TWpsNWhsZmNPelZicXBEdEZVUlNPQUY5KzdzR1Jjejc4WGZyeGFjUjY2QlhaaEs4eThkU3cxbTd5VXFBeTVyeEZpck9mVzBQNWc5dUlidERLZ0trUElCME5wRDVtTVk5bE9WaE5DTHF5RnRnT0Q0OHVuaHRvbXZ6OTEyNmZGUmVPUllSYVJORGoyZTkzTDA5OWZSb0ZRVGViTUtWUFd4ZW5cL1piSzhIWFpJZUxsV3VNcUdjQXR5dmlLQXRXNDAiLCJzaXRlX2lkIjoiZGVmYXVsdCIsImFwaSI6Im9lbXIifQ==",
+    "expires_in": "3600",
+    "user_data": {
+        "user_id": "1"
+    }
+}
+```
+For ssh calls each call must include the token:
+
+
+```sh
+curl -X GET 'http://localhost:8300/apis/fhir/Patient' \
+  -H 'Authorization: Bearer eyJ0b2tlbiI6IjAwNnZ3eGJZYmFrOXlxUjF4U290Y1g4QVVDd3JOcG5yYXZEaFlqaHFjWXJXRGNDQUtFZmJONkh2cElTVkJiaWFobHBqOTBYZmlNRXpiY2FtU01pSHk1UzFlMmgxNmVqZEhcL1ZENlNtaVpTRFRLMmtsWDIyOFRKZzNhQmxMdUloZmNJM3FpMGFKZ003OXdtOGhYT3dpVkx5b3BFRXQ1TlNYNTE3UW5TZ0dsUVdQbG56WjVxOVYwc21tdDlSQ3RvcDV3TEkiLCJzaXRlX2lkIjoiZGVmYXVsdCIsImFwaSI6ImZoaXIifQ=='
+```
+While using an API client you can include the token as part of authorization 
+
+#### GET /fhir/patient
+
+```sh
+curl -X GET 'http://localhost:8300/apis/fhir/Patient'
+```
+
+
+### Dev Notes
+
+- For business logic, make or use the services [here](https://github.com/openemr/openemr/tree/master/services)
+- For controller logic, make or use the classes [here](https://github.com/openemr/openemr/tree/master/rest_controllers)
+- For routing declarations, use the class [here](https://github.com/openemr/openemr/blob/master/_rest_routes.inc.php).
+
+
+### Project Management
+
+#### FHIR
+- TODO(?): ?
+
+### What is that dog drawing?
```

Comment:
Again, may be time to retire the logo/peppy here and in the API doc

this will likely make @robertdown very sad; but what can we do :)
(this is sarcastic, btw, since there is bad blood between peppy and @robertdown :) )